pkg/apps/alert/latest/alertservice.go

```diff
@@ -22,69 +22,46 @@ under the License.
 package alert
 
 import (
-	"fmt"
-
 	horizonapi "github.com/blackducksoftware/horizon/pkg/api"
 	"github.com/blackducksoftware/horizon/pkg/components"
 	"github.com/blackducksoftware/synopsys-operator/pkg/util"
 )
 
-// getAlertService returns a new cluster Service for an Alert
-func (a *SpecConfig) getAlertService() *components.Service {
-	service := components.NewService(horizonapi.ServiceConfig{
-		Name:      util.GetResourceName(a.alert.Name, util.AlertName, "alert"),
-		Namespace: a.alert.Spec.Namespace,
-		Type:      horizonapi.ServiceTypeServiceIP,
-	})
-
-	service.AddPort(horizonapi.ServicePortConfig{
-		Port:       int32(*a.alert.Spec.Port),
-		TargetPort: fmt.Sprintf("%d", *a.alert.Spec.Port),
-		Protocol:   horizonapi.ProtocolTCP,
-		Name:       fmt.Sprintf("%d-tcp", *a.alert.Spec.Port),
-	})
-
-	service.AddSelectors(map[string]string{"app": util.AlertName, "component": "alert"})
-	service.AddLabels(map[string]string{"app": util.AlertName, "component": "alert"})
-	return service
+// GetAlertClusterService returns a new cluster Service for an Alert
+func (a *SpecConfig) getAlertClusterService() *components.Service {
+	return util.CreateService(
+		util.GetResourceName(a.alert.Name, util.AlertName, "alert"),
+		map[string]string{"app": util.AlertName, "component": "alert"},
```

Comment:
@msenmurugan I've kept the same label and selection as previously.  But in other resources, blackduck and opssight, a name is used for a non-exposed service.  Just want to make sure we want it here in alert this way, cause otherwise I could just use `a.getLabel("alert")` here as well.

Reply:
please use getLabel. I think, I have mixed to add it here...

Reply:
Cool, done.